src/pages/menuUpload/customerMaster/CustomerMasterPage.tsx

```diff
@@ -217,6 +229,14 @@ function CustomerMasterPage() {
     }
     payload.customer_type = formValues.customer_type.id;
     dispatch(putCustomerMasterRequest(payload as PutCustomerMasterRequestPayload));
+
+    if (customerDeliveryHours) {
+      const payloadDeliveryHours = { week_days: customerDeliveryHours, id: objectToEdit.id };
+      dispatch(
+        // @ts-ignore
```

Comment:
@KarolinaKopacz Should we remove this?

Reply:
Not really, type-script despite passing on the type, doesn't see it so this is one way to pass the payload.

Reply:
ok